src/filters/defaults/blur/BlurFilter.ts

```diff
@@ -209,20 +212,6 @@ export class BlurFilter extends Filter
         this.updatePadding();
     }
 
-    /**
-     * Sets the blendmode of the filter
-     * @default "normal"
-     */
-    get blendMode(): BLEND_MODES
-    {
-        return this.blurYFilter.blendMode;
-    }
-
-    set blendMode(value: BLEND_MODES)
-    {
-        this.blurYFilter.blendMode = value;
-    }
-
```

Comment:
Also fixes a bug, I think: In the x-blur-only case the blend mode wouldn't have been applied previously.